com.endlessm.apps.Sdk.json.in

```diff
@@ -132,7 +132,7 @@
         {
             "name": "tcl",
             "no-autogen": true,
-            "cleanup": ["*"],
+            "cleanup-platform": ["*"],
```

Comment:
Ugh… This is kind of an indication that we should probably *not* use the SDK for doing CI on the SDK itself, but I have no better solutions. Getting TCL in the SDK will increase its size a bit.